src/Utils/Data/Calculations.java

```diff
@@ -33,11 +33,15 @@ public static void runThreadedCalculation() {
 
             convertCurrencies(GUI.getBaseCur(), GUI.getTargetCur(), GUI.getAmount());
 
-            GUI.setOutput("Eingetippt: " + GUI.getAmount() + " " + GUI.getBaseCur() + "\n" +
-                    "Das Ergebnis ist " + finalResult + " " + GUI.getTargetCur() + "\n" +
-                    "Wechselkurs: " + ExchangeRateFetcher.getLatestExchangeRate() + "\n" +
-                    "Wechselkurs herausgefunden in " + ExchangeRateFetcher.getLastFetchTime() + "ms");
-
+            if (Utils.failed == false) {
+                GUI.setOutput("Eingetippt: " + GUI.getAmount() + " " + GUI.getBaseCur() + "\n" +
+                        "Das Ergebnis ist " + finalResult + " " + GUI.getTargetCur() + "\n" +
+                        "Wechselkurs: " + ExchangeRateFetcher.getLatestExchangeRate() + "\n" +
+                        "Wechselkurs herausgefunden in " + ExchangeRateFetcher.getLastFetchTime() + "ms");
+            } else {
+                GUI.setOutput("");
+                Utils.failed = false;
```

Comment:
können wir hier vllt ne methode draus machen? Und das ganze auch nach ExchangeRateFetcher packen? Dann kannst du in ExchangeRateFetcher ne nen private static final boolean daraus machen und ne funktion namens ExchangeRateFetcher.hasFailed() die den boolean zurück gibt? Ist glaube ich cleaner